dataprofiler/profilers/float_column_profile.py

```diff
@@ -148,12 +217,56 @@ def _update_precision(self, df_series, prev_dependent_properties,
         :type df_series: pandas.DataFrame
         :return: None
         """
+
+        # (min, max, var, sum, sample_size)
         subset_precision = self._get_float_precision(df_series)
-        if self.precision and subset_precision:
-            self.precision = min(self.precision, subset_precision)
-        elif subset_precision:
-            self.precision = subset_precision
+        if subset_precision is None: return
+        
+        # Update values
+        if self.precision['min'] is None:
```

Comment:
if any one of these values are `None` for precision, can we not assume the rest are also `None`?
e.g.
```python
if self.precision['min'] is None:
    self.precision = subset_precision
    return
self.precision['min']  = min(self.precision['min'], subset_precision['min'])
self.precision['max']  = min(self.precision['max'], subset_precision['max'])
self.precision['sum']  += subset_precision['sum'])
self.precision['var'] = self._merge_variance(...)
.
.
.
```

Reply:
similarly, this code and the code above in the `__add__` are near identical

Reply:
I agree, in `__add__` if you use just a check of `['min']` being none to determine whether or not precision has been calculated it feels fair to do the same here.